pkg/installation/move.go

```diff
@@ -196,18 +196,19 @@ func moveOrCopyDir(from, to string) error {
 		glog.V(4).Infof("Move target directory %q cleaned up", to)
 	}
 
-	err = os.Rename(from, to)
+	err = copy(from, to)
```

Comment:
This basically means, that we don't even try to rename files anymore, right? This is a bit sad. I wish there were a better way.

If I'm correct, then we don't need the rename-specific error handling below.

Reply:
Hmm wait this is weird. I think we need to keep this as `os.Rename`, so we run into the LinkError below. I'll update, but likely can't test right now.

Reply:
Fixed.